contributor-docs/release-guide.md

```diff
@@ -1228,10 +1228,7 @@ __NOTE__: This can only be done from `@apache.org` email address. This email has
 
 ### Social media
 
-Tweet, post on Facebook, LinkedIn, and other platforms.
-Ask other contributors to do the same.
```

Comment:
I think we should still do this with some new features.

Reply:
How would you frame then? At the very least, it needs changed since most release managers don't have permissions to do this.

In general, I think tweeting about releases does not get us a whole lot

Reply:
I modified to just suggest linkedin based on offline conversation. I also moved it to the PMC section